pkg/rehearse/jobs_test.go

```diff
@@ -529,72 +558,54 @@ func TestExecuteJobsPositive(t *testing.T) {
 	}{{
 		description: "two jobs in a single repo",
 		jobs: map[string][]prowconfig.Presubmit{targetRepo: {
-			*makeTestingPresubmit("job1", "ci/prow/job1", []string{"arg1"}, "master"),
-			*makeTestingPresubmit("job2", "ci/prow/job2", []string{"arg1"}, "master"),
+			*makeTestingPresubmit("job1", "ci/prow/job1", targetOrg, targetRepo, "master"),
+			*makeTestingPresubmit("job2", "ci/prow/job2", targetOrg, targetRepo, "master"),
 		}},
 		expectedJobs: []pjapi.ProwJobSpec{
 			makeTestingProwJob(testNamespace,
 				"rehearse-123-job1",
 				fmt.Sprintf(rehearseJobContextTemplate, targetRepo, "master", "job1"),
-				testRefs,
-				[]string{"arg1", fmt.Sprintf("--git-ref=%s@master", targetRepo)},
-			).Spec,
+				testRefs, targetOrg, targetRepo, "master").Spec,
 			makeTestingProwJob(testNamespace,
 				"rehearse-123-job2",
 				fmt.Sprintf(rehearseJobContextTemplate, targetRepo, "master", "job2"),
-				testRefs,
-				[]string{"arg1", fmt.Sprintf("--git-ref=%s@master", targetRepo)},
-			).Spec,
+				testRefs, targetOrg, targetRepo, "master").Spec,
 		}}, {
 		description: "two jobs in a single repo, same context but different branch",
 		jobs: map[string][]prowconfig.Presubmit{targetRepo: {
-			*makeTestingPresubmit("job1", "ci/prow/job1", []string{"arg1"}, "master"),
-			*makeTestingPresubmit("job2", "ci/prow/job2", []string{"arg1"}, "not-master"),
+			*makeTestingPresubmit("job1", "ci/prow/job1", targetOrg, targetRepo, "master"),
+			*makeTestingPresubmit("job2", "ci/prow/job2", targetOrg, targetRepo, "not-master"),
 		}},
 		expectedJobs: []pjapi.ProwJobSpec{
 			makeTestingProwJob(testNamespace,
 				"rehearse-123-job1",
 				fmt.Sprintf(rehearseJobContextTemplate, targetRepo, "master", "job1"),
-				testRefs,
-				[]string{"arg1", fmt.Sprintf("--git-ref=%s@master", targetRepo)},
-			).Spec,
+				testRefs, targetOrg, targetRepo, "master").Spec,
 			makeTestingProwJob(testNamespace,
 				"rehearse-123-job2",
 				fmt.Sprintf(rehearseJobContextTemplate, targetRepo, "not-master", "job2"),
-				testRefs,
-				[]string{"arg1", fmt.Sprintf("--git-ref=%s@not-master", targetRepo)},
-			).Spec,
+				testRefs, targetOrg, targetRepo, "not-master").Spec,
 		}},
 		{
 			description: "two jobs in a separate repos",
 			jobs: map[string][]prowconfig.Presubmit{
-				targetRepo:        {*makeTestingPresubmit("job1", "ci/prow/job1", []string{"arg1"}, "master")},
-				anotherTargetRepo: {*makeTestingPresubmit("job2", "ci/prow/job2", []string{"arg1"}, "master")},
+				targetRepo:        {*makeTestingPresubmit("job1", "ci/prow/job1", targetOrg, targetRepo, "master")},
```

Comment:
Shouldnt keys for presubmits be `org/repo` format?

Reply:
those are being used only to fill the `ExtraRefs`

Reply:
Right but you're using it as the key, isn't that weird? Why?

Reply:
ah.. sry I misunderstood you above. Yea it seems that we don't care for the key much, but you are right we need to be consistent here. I will change this.

Reply:
addressed